seshdash/static/sesh-ui/js/time_series_graph.js

```diff
@@ -0,0 +1,126 @@
+
+var csrftoken = getCookie('csrftoken');
+
+var measurement_default_value;
+var time_default_value;
+$(document).ready(function(){
+
+    measurement_value = $('#measurements_dropdown').val();
+    time_default_value = $('#time_dropdown').val();
+
+    $.post('/time_series',{csrfmiddlewaretoken:csrftoken,'measurement':measurement_value,'time':time_default_value,'active_id':active_site_id},function(data){
+                 var data_values = JSON.parse(data)
+                 si_units = data_values.units;
+                 graph_data_values=data_values.graph_values;
+
+
+                  for(var i =0 , l= graph_data_values.length ; i<l ;i++){
+                      graph_data_values[i][0] = graph_data_values[i][0] * 1000;
+                 }
+                  
+            
+         $('#time_series_graph').highcharts({
```

Comment:
This time series graph code is repeated too often. We should have a function or a seperate javascript that creates this automatically